ton_client/client/src/tests/mod.rs

```diff
@@ -253,10 +268,9 @@ impl TestClient {
     }
 
     pub fn network_address() -> String {
-        std::env::var("TON_NETWORK_ADDRESS")
-            .unwrap_or("http://localhost".to_owned())
-            //.unwrap_or("cinet.tonlabs.io".to_owned())
-            //.unwrap_or("net.ton.dev".to_owned())
+        std::env::var("TON_NETWORK_ADDRESS").unwrap_or("http://localhost".to_owned())
```

Comment:
Return previous formatting. It is used for easy network changing

Reply:
introduce constants with pretty well formatting